src/ir/utils.h

```diff
@@ -300,7 +298,7 @@ struct AutoDrop : public WalkerPass<ExpressionStackWalker<AutoDrop>> {
   void doWalkFunction(Function* curr) {
     ReFinalize().walkFunctionInModule(curr, getModule());
     walk(curr->body);
-    if (curr->result == none && curr->body->type.isConcrete()) {
+    if (curr->sig.results == Type::none && curr->body->type.isConcrete()) {
```

Comment:
I would still like to land #2434, but since I started this before we came to consensus on that, I am just adding `Type::` on any line that this PR changes anyway. That will make rebasing and landing #2434 easier afterwards.

Reply:
So your suggestion is basically wherever we add new code we append `Type::`?